### ISSUE TITLE
It is possible to close an alert dynamically

### ISSUE THREAD

**Issue by hiniesta** [RatelHub/rflutter_alert#28]
First at all, grat job!
I will like to close an alert via a timeout function. It is posible?

**Comment by RafikFarhad** [comment]
This feature will be awesome for this awesome package.

**Comment by RafikFarhad** [comment]
@hiniesta 
You can achieve this by the following code snippet.

```
Alert(context: context, title: "RFLUTTER", desc: "Flutter is awesome.").show();

Timer(Duration(seconds: 3), () {
      Navigator.of(context).pop();
});

**Comment by va0000ll** [comment]
> 
> 
> @hiniesta
> You can achieve this by the following code snippet.
> 
> ```
> Alert(context: context, title: "RFLUTTER", desc: "Flutter is awesome.").show();
> 
> Timer(Duration(seconds: 3), () {
>       Navigator.of(context).pop();
> });
> ```

Does not work for me ... it just pop up my in my app route and Alert still there .

**Comment by HugoHeneault** [comment]
See https://github.com/RatelHub/rflutter_alert/issues/20 :)

**Comment by ibrahimdevs** [comment]
@hiniesta thank you for using this project. Can you try @RafikFarhad solution again. it should be fixed on V1.0.6.
